### DISCUSSION TITLE
Error 400

### DISCUSSION THREAD

**Discussion by LorealEndPoint** [almenscorner/Munki-Manifest-Generator#7] [Q&A]
Hey People, 

was in the version 1.0 and just updated to the V1.2 because the V1 is not managing the renew of token. 
I've now a new error, and don't know from who it arrive. 
[ERROR] Request failed with status code 400 for 5fde1680-90b8-45c9-a61c-xxxxxxx29e350cc (concurrent_batch.py:153:get_response_body)
we can see that the generator is checking the ID of device. but there is an error after. 

From what i saw it seems that the ID in the line is Object ID (which is not present in Intune) and not a Device ID or Intune ID. 

so I've got for all the manifest update the same error, and NO creation of manifest. 

do you have an idea to solve that issue?

**Comment by LorealEndPoint** [comment]
so by checking I can see in the code that you are reporting: azureADDeviceId
That value is not known by Intune.
You should probable change it by "id" which is known is entra and Intune

**Reply by almenscorner** [reply]
`azureADDeviceId` is part of the object when you call `https://graph.microsoft.com/v1.0/deviceManagement/managedDevices` because the ID of the object in Entra is different from the ID in Intune so I don't think that is the issue as the generator is running fine for me. What is the output if you enable verbose output?

**Reply by LorealEndPoint** [reply]
thanks for your answer, 

On my AZ automation it is already activated, but not enought logs I guess....
I will try to launch everything on my Mac directly

**Comment by almenscorner** [comment]
You can use `log=DEBUG` in the runbook

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Reply by LorealEndPoint** [reply]
Hey TObias, 
by adding : if webhook is True:
    mmg.main(group_list=groups, serial_number=serial, log=DEBUG)
else:
    mmg.main(group_list=groups, log=DEBUG)
in the webhook I do not have anymore message, only:

No errors found for the job
--

**Reply by almenscorner** [reply]
So did the manifest generate/update?

**Reply by LorealEndPoint** [reply]
no :( 
just rerun it and seen that message: 
Traceback (most recent call last): File "/usr/src/tmp/8a7f0eb2-4efc-470f-bf33-b3ca382f769e/runbooks/MunkiManifest1.py", line 786, in <module> mmg.main(group_list=groups, log=DEBUG) NameError: name 'DEBUG' is not defined

**Reply by almenscorner** [reply]
You need to set it as `log="DEBUG"`, sorry about that
